interfaces/mount/change.go

```diff
@@ -42,6 +44,29 @@ type Change struct {
 	Action Action
 }
 
+// Perform executes the desired mount or unmount change.
```

Comment:
We import code from all over the tree. I think it is okay to keep it here but if you feel strongly about it I can move it elsewhere.